### DISCUSSION TITLE
setShrinkToFit

### DISCUSSION THREAD

**Discussion by thorhammeren** [PHPOffice/PhpSpreadsheet#2017] [Q&A] [answered]
Hi,

Where to put setShrinkToFit() ?

$sheet = $spreadsheet->getActiveSheet();
I set $sheet->getCell("B3")->setShrinkToFit(true);

And i get error :

Fatal error: Call to undefined method PhpOffice\PhpSpreadsheet\Worksheet\Worksheet::setShrinkToFit()

**Comment by oleibman** [comment, marked as answer]
```
$sheet()->getStyle('B3')->getAlignment()->setShrinkToFit(true);
```
